cluster/src/main/java/org/apache/iotdb/cluster/rpc/raft/processor/querydata/QuerySeriesDataSyncProcessor.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.cluster.rpc.raft.processor.querydata;
+
+import com.alipay.remoting.BizContext;
+import com.alipay.sofa.jraft.Status;
+import org.apache.iotdb.cluster.config.ClusterConstant;
+import org.apache.iotdb.cluster.query.PathType;
+import org.apache.iotdb.cluster.query.manager.querynode.ClusterLocalQueryManager;
+import org.apache.iotdb.cluster.rpc.raft.processor.BasicSyncUserProcessor;
+import org.apache.iotdb.cluster.rpc.raft.request.querydata.QuerySeriesDataRequest;
+import org.apache.iotdb.cluster.rpc.raft.request.querydata.Stage;
+import org.apache.iotdb.cluster.rpc.raft.response.querydata.QuerySeriesDataResponse;
+import org.apache.iotdb.cluster.utils.RaftUtils;
+import org.apache.iotdb.db.exception.ProcessorException;
+
+public class QuerySeriesDataSyncProcessor extends
+    BasicSyncUserProcessor<QuerySeriesDataRequest> {
+
+  @Override
+  public Object handleRequest(BizContext bizContext, QuerySeriesDataRequest request)
+      throws Exception {
+    Stage stage = request.getStage();
+    String groupId = request.getGroupID();
+    QuerySeriesDataResponse response = new QuerySeriesDataResponse(groupId);
+    switch (stage) {
+      case INITIAL:
+        handleNullRead(request.getReadConsistencyLevel(), groupId);
+        ClusterLocalQueryManager.getInstance().createQueryDataSet(request, response);
```

Comment:
Can we make this method return `response` rather than pass `response` into this method ? Maybe returning response by this method seems more straightforward.

Reply:
accept the advice. fixed